doc/releases/changelog-dev.md

```diff
@@ -8,6 +8,9 @@
   that produces a set of gate names to be used as the target gate set in decompositions.
   [(#8522)](https://github.com/PennyLaneAI/pennylane/pull/8522)
 
+* Added a :func:`~pennylane.measurements.pauli_measure` that takes a Pauli product measurement.
+  [(#8461)](https://github.com/PennyLaneAI/pennylane/pull/8461)
+
```

Comment:
Hi @astralcai I just want to leave a note here that an example should be added to this changelog entry.
Is this something that would be done latter?

Reply:
Yes. There's a separate story tracking all the documentation work, which would also include a more comprehensive changelog section about PPMs